src/moin/cli/_tests/test_modify_item.py

```diff
@@ -50,10 +52,6 @@ def test_dump_help(load_help):
         # data_file_names = set([p.name for p in help_subdir_path.glob('*.data')])
         # assert expected_data_file_names == data_file_names
         for data_file_name in expected_data_file_names:
-            # TODO remove next 3 lines - adding new help-en items causes test failure
```

Comment:
could not reproduce this test failure

Reply:
after merge and pull into master, the problem went away